identifiers/country-tw/centrally_governed_level.csv

```diff
@@ -0,0 +1,25 @@
+id,name
+ocd-division/country:tw/county:cha,Changhua
+ocd-division/country:tw/county:cyq,Chiayi
+ocd-division/country:tw/county:hsq,Hsinchu
+ocd-division/country:tw/county:hua,Hualien
+ocd-division/country:tw/county:kin,Kinmen
+ocd-division/country:tw/county:lie,Lienchiang
+ocd-division/country:tw/county:mia,Miaoli
+ocd-division/country:tw/county:nan,Nantou
+ocd-division/country:tw/county:pen,Penghu
+ocd-division/country:tw/county:pif,Pingtung
+ocd-division/country:tw/county:ttt,Taitung
+ocd-division/country:tw/county:ila,Yilan
+ocd-division/country:tw/county:yun,Yunlin
+ocd-division/country:tw/city:cyi,Chiayi
```

Comment:
in other countries we've used "place" instead of "city". Let's try to be consistent and do that.